amx/server/natives/a_objects.lua

```diff
@@ -34,24 +34,28 @@ function SetObjectPos(amx, object, x, y, z)
 		setElementVelocity(object, 0, 0, 0)
 		setTimer(setElementFrozen, 500, 1, object, false)
 	end
+
+	return true
 end
 
 function GetObjectPos(amx, object, refX, refY, refZ)
 	local x, y, z = getElementPosition(object)
 	writeMemFloat(amx, refX, x)
 	writeMemFloat(amx, refY, y)
 	writeMemFloat(amx, refZ, z)
+	return true
 end
 
 function GetObjectRot(amx, object, refX, refY, refZ)
 	local rX, rX, rZ = getObjectRotation(object)
 	writeMemFloat(amx, refX, rX)
 	writeMemFloat(amx, refY, rY)
 	writeMemFloat(amx, refZ, rZ)
+	return true
 end
 
 function SetObjectRot(amx, object, rX, rY, rY)
-	setObjectRotation(object, rX, rY, rZ)
+	return setObjectRotation(object, rX, rY, rZ)
```

Comment:
Apparently this should always return `true` for compatibility reasons, even if said object doesn't exist.